esp-hal/src/lcd_cam/lcd/i8080.rs

```diff
@@ -336,11 +327,11 @@ where
     }
 
     /// Sends a command and data to the LCD using the I8080 interface.
-    pub fn send(
+    pub fn send<W: Copy + Into<u16>>(
```

Comment:
Let's document that you have to choose a data type, and what possible behaviours can happen when using a different data type compared to how the bus is set up.

We probably need this or a reference to it on all the `pub send*` functions.

Reply:
How about we split this in two, and provide `send_8` and `send_16` instead?

Reply:
I did think about that, but we would end up with 3 extra functions for all the variants of `send`, it seems like we should try and let the compiler do the monomorphization for us here - but maybe ambiguous type errors are enough to warrant this :thinking:.

Reply:
In my next PR I'll be reducing the 3 methods to just one. I can resolve the ambiguity there if you like. (I'll add docs to this PR anyway)